homework/src/style.css

```diff
@@ -1,3 +1,109 @@
 .alert-error {
     color: red;
+    text-align: center;
```

Comment:
Your application is not responsive. This is what it looks like on an iPhone:

![screenshot 2018-11-07 at 15 24 54](https://user-images.githubusercontent.com/2788771/48137046-7415d500-e2a1-11e8-997e-5b4c9a0d0671.png)